docs/api/plugins/aea_ledger_cosmos/cosmos.md

```diff
@@ -952,6 +952,22 @@ Get all transfer events derived from a transaction.
 - `tx_hash`: the transaction hash
 - `target_address`: optional address to filter tranfer events to just those that affect it
 
+<a id="plugins.aea-ledger-cosmos.aea_ledger_cosmos.cosmos._CosmosApi.bundle_and_send"></a>
+
+#### bundle`_`and`_`send
+
+```python
+def bundle_and_send(raw_signed_transactions: List[str],
+                    target_blocks: List[int]) -> Optional[List[str]]
+```
+
+Simulate and send a bundle of transactions.
+
+**Arguments**:
```

Comment:
Not sure how it is for other cases, but should this say that this operation is not supported for cosmos?

Reply:
Done in 94b96880e7a65eba064b81727620a07a530d5957.